app/src/main/java/com/weatherxm/ui/common/UIModels.kt

```diff
@@ -101,7 +110,7 @@ data class UIDevice(
         return isOwned()
             && !currentFirmware.equals(assignedFirmware)
             && !assignedFirmware.isNullOrEmpty()
-            && profile == DeviceProfile.Helium
+            && (bundleName == BundleName.h1 || bundleName == BundleName.h2)
```

Comment:
Can we use the `isHelum()` here?

Reply:
We can, but `isHelium` refers to connectivity whereas by checking against `BundleName` is better I think due to the fact that it's more "straightforward". As in order to update a station through app we should check whether this station supports it, not whether the connectivity supports it.